recipes/cbflib/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "CBFlib" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/yayahjb/cbflib/archive/refs/tags/CBFlib-{{ version }}.tar.gz
+    folder: cbflib
+    sha256: e4acf8f7b5cc4e788215983ff428c2f8aa1fdbbfc6d163ace7999dcd2a10b19f
+    patches:
+      - 01-off-by-one.patch
+      - 02-overrun.patch
+      - 03-binary-size-third-dimension-2D.patch
+      - 04-fgetln.patch
+      - 05-mkstemplate.patch
+      - 06-realpath.patch
+      - 07-vla.patch
+      - 08-libtiff-public-api.patch
+      - 09-pycbf-dials.patch
+      - 10-cmake.patch
+      - 11-cmakelist.patch
+      - 12-msvc.patch
+      - 13-warnings.patch
+  - url: http://downloads.sf.net/cbflib/CBFlib_{{ version }}_Data_Files_Input.tar.gz
+    sha256: 5d9eac1adb1a49dbc3a65455cd631e3a1849fe3c07301e5ec8be9f3c42da8025
+    folder: data-input
+    patches:
+      - cbflib-data-input-{{ version }}.patch
+  - url: http://downloads.sf.net/cbflib/CBFlib_{{ version }}_Data_Files_Output.tar.gz
+    sha256: 06c9e98410b39c5f40cfce62a940857172d5914b172dca8cf15a69b330d6fec6
+    folder: data-output
+    patches:
+      - cbflib-data-output-{{ version }}.patch
+
+build:
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}      # [win]
```

Comment:
Why?

Reply:
This is to silence overlinking warnings from `conda-build`; they can be removed without breaking the build but if I interpret the output correctly, they're not actually used to resolve any symbols.